ui/components/app/transaction-activity-log/transaction-activity-log-icon/transaction-activity-log-icon.component.js

```diff
@@ -31,24 +33,16 @@ export default class TransactionActivityLogIcon extends PureComponent {
 
   static propTypes = {
     className: PropTypes.string,
-    eventKey: PropTypes.oneOf(Object.keys(imageHash)),
+    eventKey: PropTypes.oneOf(Object.keys(ACTIVITY_ICONS)),
   };
 
   render() {
     const { className, eventKey } = this.props;
-    const iconClassName = imageHash[eventKey];
+    const icon = ACTIVITY_ICONS[eventKey];
 
     return (
       <div className={classnames('transaction-activity-log-icon', className)}>
-        {iconClassName ? (
-          <i
-            className={classnames(
-              'fa',
-              'transaction-activity-log-icon__icon',
-              iconClassName,
-            )}
-          />
-        ) : null}
+        {icon ? <Icon name={icon} color={Color.iconDefault} /> : null}
       </div>
     );
   }
```

Comment:
After


https://user-images.githubusercontent.com/8112138/217699598-778f4ec7-534d-4f09-9b5c-f05b64b1176a.mov

Reply:
I updated the size here; looking at the code, do any of the icons look completely wrong?